src/vis/pcp/utils.ts

```diff
@@ -20,12 +22,13 @@ const defaultConfig: IPCPConfig = {
 };
 
 export function pcpMergeDefaultConfig(
-    columns: (NumericalColumn | CategoricalColumn)[],
+    columns: VisColumn[],
     config: IPCPConfig,
 ): IVisConfig {
     const merged = merge({}, defaultConfig, config);
 
     if(merged.numColumnsSelected.length === 0 && columns.length > 1) {
+        // TODO: Bug. It is always selecting the last two columns, no matter their type.
```

Comment:
If this is a bug, please open an issue and provide some information how to reproduce the error, the observed and expected behavior. Afterwards you can also add the issue link here, so that (other) developers have some context for this bug. The issue also helps to schedule a possible fix. Thanks.

Reply:
added an issue and changed to FIXME. The issue is here: https://github.com/datavisyn/reprovisyn/issues/199